ec2-cluster/modules/node/templates/restart

```diff
@@ -5,6 +5,6 @@ if [[ $EUID -ne 0 ]]; then
   exit 1
 fi
 
-mv -v /home/admin/tessellation/logs /home/admin/tessellation/logs_$(date +"%Y-%m-%dT%H:%M:%SZ") 2>/dev/null
-/home/admin/tessellation/update-seedlist
+mv -v /home/admin/tessellation/l0/logs /home/admin/tessellation/l0/logs_$(date +"%Y-%m-%dT%H:%M:%SZ") 2>/dev/null
+/home/admin/tessellation/l0/update-seedlist
```

Comment:
in the `update-seedlist` file you need to provide the full path to wget output parameter. otherwise the script will fetch the file but into wrong place.

Reply:
<img width="1372" alt="image" src="https://user-images.githubusercontent.com/9077743/188117217-def4a4ae-8b1b-4d41-842d-48ca870add9d.png">

The script will output the wget in the directory of script execution so it is fine unless we execute it by another script 🤔 
But I can change it if you want

Reply:
it works when you execute it manually but it does not work when you execute it from the restart script from first node

Reply:
Changed :)